coffee/expression.py

```diff
@@ -155,16 +172,11 @@ def mode(self, value):
 def copy_metaexpr(expr_info, **kwargs):
     """Given a ``MetaExpr``, return a plain new ``MetaExpr`` starting from a
     copy of ``expr_info``, and replaces some attributes as specified in
-    ``kwargs``. In particular, ``kwargs`` has the following keys:
-
-        * parent: the block node that embeds the expression.
-        * loops_info: an iterator of 2-tuple ``(loop, loop_parent)`` which
-          substitute analogous information in the new ``MetaExpr``.
-        * domain: the domain of the output tensor evaluated by the expression.
-        * mode: the suggested rewrite mode.
-    """
+    ``kwargs``. ``kwargs`` accepts the following keys: parent, loops_info,
+    linear_dims, mode."""
```

Comment:
it was basically the same description as the one you can find in the MetaExpr class' docstring